sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactImportExportSteps.java

```diff
@@ -0,0 +1,220 @@
+        () -> webDriverHelpers.clickOnWebElementBySelector(NEW_EXPORT_CONFIGURATION_BUTTON));
+
+    When(
+        "I fill Configuration Name field with {string} in Custom Contact Export popup",
+        (String confName) -> {
+          configurationName = confName + LocalDate.now().toString();
+          webDriverHelpers.fillInWebElement(CONFIGURATION_NAME_INPUT, configurationName);
+        });
+
+    When(
+        "I select specific data to export in Export Configuration for Custom Contact Export",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(EXPORT_CONFIGURATION_DATA_DISEASE_CHECKBOX);
+          webDriverHelpers.clickOnWebElementBySelector(
+              EXPORT_CONFIGURATION_DATA_ID_CHECKBOX_CONTACT);
+          webDriverHelpers.clickOnWebElementBySelector(
+              EXPORT_CONFIGURATION_DATA_FIRST_NAME_CHECKBOX_CONTACT);
+          webDriverHelpers.clickOnWebElementBySelector(
+              EXPORT_CONFIGURATION_DATA_LAST_NAME_CHECKBOX_CONTACT);
+          webDriverHelpers.clickOnWebElementBySelector(NEW_EXPORT_CONFIGURATION_SAVE_BUTTON);
+        });
+
+    When(
+        "I download created custom contact export file",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(CUSTOM_CASE_EXPORT_DOWNLOAD_BUTTON);
+          TimeUnit.SECONDS.sleep(5); // wait for download
+        });
+
+    When(
+        "I check if downloaded data generated by custom contact option is correct",
+        () -> {
+          String file =
+              "./downloads/sormas_kontakte_" + LocalDate.now().format(formatter) + "_.csv";
+          CustomContactExportCSV reader = parseCustomContactExport(file);
+          Path path = Paths.get(file);
+          Files.delete(path);
+          softly.assertEquals(
+              reader.getUuid(), apiState.getCreatedContact().getUuid(), "UUIDs are not equal");
+          softly.assertEquals(
+              reader.getDisease(),
+              DiseasesValues.getCaptionForName(apiState.getCreatedContact().getDisease()),
+              "Diseases are not equal");
+          softly.assertEquals(
+              String.format(reader.getFirstName(), Locale.GERMAN),
+              String.format(apiState.getLastCreatedPerson().getFirstName(), Locale.GERMAN),
+              "First names are not equal");
+          softly.assertEquals(
+              String.format(reader.getLastName(), Locale.GERMAN),
+              String.format(apiState.getLastCreatedPerson().getLastName(), Locale.GERMAN),
+              "Last names are not equal");
+          softly.assertAll();
+        });
+
+    When(
+        "I check if downloaded data generated by basic contact option is correct",
+        () -> {
+          String file =
+              "./downloads/sormas_kontakte_" + LocalDate.now().format(formatter) + "_.csv";
+          Contact reader = parseBasicContactExport(file);
+          Path path = Paths.get(file);
+          Files.delete(path);
+          softly.assertEquals(
```

Comment:
here, you can do the assert with poll 20s, and by doing this, you can remove the sleep added in the method where you click to download.